pkgs/development/libraries/bzrtp/default.nix

```diff
@@ -7,7 +7,7 @@
 
 stdenv.mkDerivation rec {
   pname = "bzrtp";
-  version = "4.4.0";
+  version = "4.4.9";
```

Comment:
I think your forgot to change the hash here.

Reply:
Don't ask me why, but `1yjmsbqmymzl4r7sba6w4a2yld8m6hzafr6jf7sj0syhwpnc3zv6` is indeed the correct hash for bzrtp-4.4.9, I tried changing it but that is what got fetched.

Reply:
thats weird.